translator/translate/otel/processor/jmxprocessor/translator.go

```diff
@@ -0,0 +1,106 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package jmxprocessor
+
+import (
```

Comment:
nit: Fix your import grouping and run `make fmt simple-lint`.

Reply:
Done, fixed and grouped